### ISSUE TITLE
[Feature request]: nes Famicom virtual button position results

### ISSUE THREAD

**Issue by soundsnow** [Rakashazi/emu-ex-plus-alpha#1331]
### Describe the feature

Family Basic uses two Gamepads because it supports P2 Microphone. Gamepads are bundled with Famicom, which means that when you plug in other input devices, you need to use one or two Gamepads to select and start the game

The button positions of the P1 and P2 Gamepads can be swapped, or hide one of them
Old Famicom P2 Gamepad with Microphone, without Select and Start.The new Famicom P2 Gamepads are like the P1, you can just unplug them and plug them into other input devices.
Virtual button positions for two-player expansion interface input devices can also be swapped for compatible games. For example, the Konami HyperShot controllers indicate that they support two players
Virtual button positions for the same extended input device can be interchanged.  The virtual button positions of P1 and P2 are interchangeable.  Virtual button positions for Gamepads can be swapped with virtual button positions for extended input devices
Gamepads can be hidden or shown when one or more expansion input devices are plugged in
Display or only one Gamepad and one expansion input device are allowed
In short, they can be arbitrarily combined and arranged.  If you still put Select and Start on the sides in landscape mode, the second Gamepad won't have enough room on the screen. Then it competes with the extended input device for space in the middle of the screen

**Comment by soundsnow** [comment]
I guess you didn't understand what I meant, forget it!  I am also tired.  Thanks to my hateful country for turning me from an inventor to a fool.  If I hadn't been exposed to so much sleazy stuff, if I'd stopped giving honesty and help to everyone who was greedy, hypocritical, misusing my knowledge, maybe I would have helped you improve your code for this, pity lazy and stupid  People only hurt me.  Sorry I shouldn't have complained, please don't reply me about this.  In my country there are only fake programmers who advertise a lot of crap and make rogue software.  The open source code stolen from foreign countries is said to all citizens to do it by themselves.  I don't want to be one of those people so my every day is miserable and I've never lied to anyone bad

**Comment by soundsnow** [comment]
Thank you for your long-term contribution to emulation on Android, although it has little to do with me.
